okta/src/main/java/org/ccci/idm/user/okta/dao/listeners/FallbackDaoOktaUserDaoListener.kt

```diff
@@ -31,6 +31,9 @@ class FallbackDaoOktaUserDaoListener(
             // SQ & SA
             user.securityQuestion = securityQuestion
             user.securityAnswer = securityAnswer
+
+            // Login Time (fallback if there isn't already a last login time)
+            user.loginTime = user.loginTime ?: loginTime
```

Comment:
So, in this case, `user.loginTime` is from Okta and `loginTime` is from The Key?

Reply:
yes, `user` is initially generated from Okta and the passed to this method to populate any attributes that still exist in eDirectory